EasyPost.Tests/UtilitiesTests/AttributesTest.cs

```diff
@@ -18,29 +85,61 @@ public class AttributesTest
         public void TestGetMethodsWithAttributes()
         {
             // via type
-            IEnumerable<MethodInfo> methods = BaseCustomAttribute.GetMethodsWithAttribute<CrudOperations.Update>(typeof(Shipment));
+            IEnumerable<MethodInfo> methods = BaseCustomAttribute.GetMethodsWithAttribute<ExampleAttribute>(typeof(ExampleClass));
             Assert.True(methods.Any());
 
             // via instance
-            Shipment shipment = new();
-            methods = BaseCustomAttribute.GetMethodsWithAttribute<CrudOperations.Update>(shipment);
+            ExampleClass exampleClass = new();
+            methods = BaseCustomAttribute.GetMethodsWithAttribute<ExampleAttribute>(exampleClass);
             Assert.True(methods.Any());
         }
 
+        [Fact]
+        [Testing.Logic]
+        public void TestCanGetNonPublicAndStaticMethodsWithAttributes()
+        {
+            // via type
+            IEnumerable<MethodInfo> methods = BaseCustomAttribute.GetMethodsWithAttribute<ExampleAttribute>(typeof(ExampleClass));
+            // we should have 8 methods with the attribute
+            Assert.True(methods.Count() == 8);
+
+            // via instance
+            ExampleClass exampleClass = new();
+            methods = BaseCustomAttribute.GetMethodsWithAttribute<ExampleAttribute>(exampleClass);
+            // we should have 8 methods with the attribute
+            Assert.True(methods.Count() == 8);
+        }
+
         [Fact]
         [Testing.Function]
         public void TestGetPropertiesWithAttributes()
         {
             // via type
-            IEnumerable<PropertyInfo> properties = BaseCustomAttribute.GetPropertiesWithAttribute<JsonPropertyAttribute>(typeof(Shipment));
+            IEnumerable<PropertyInfo> properties = BaseCustomAttribute.GetPropertiesWithAttribute<ExampleAttribute>(typeof(ExampleClass));
             Assert.True(properties.Any());
 
             // via instance
-            Shipment shipment = new();
-            properties = BaseCustomAttribute.GetPropertiesWithAttribute<JsonPropertyAttribute>(shipment);
+            ExampleClass exampleClass = new();
+            properties = BaseCustomAttribute.GetPropertiesWithAttribute<ExampleAttribute>(exampleClass);
             Assert.True(properties.Any());
         }
 
+        [Fact]
+        [Testing.Logic]
+        public void TestCanGetNonPublicAndStaticPropertiesWithAttributes()
```

Comment:
suggestion: Please annotate tests. This looks identical on the surface to the previous test. Why/how is it different